DetDescr/include/DetDescr/DetectorIDBindings.h

```diff
@@ -0,0 +1,22 @@
+
+#ifndef DETDESCR_DETECTORID_BINDINGS_H_
+#define DETDESCR_DETECTORID_BINDINGS_H_
+
+#if DETECTORID_BINDINGS_ENABLED
+#include <boost/python.hpp>
+#endif
+#include "DetDescr/DetectorID.h"
+#include "DetDescr/EcalAbstractID.h"
+#include "DetDescr/EcalElectronicsID.h"
+#include "DetDescr/EcalID.h"
+#include "DetDescr/EcalTriggerID.h"
+#include "DetDescr/HcalAbstractID.h"
+#include "DetDescr/HcalDigiID.h"
+#include "DetDescr/HcalElectronicsID.h"
+#include "DetDescr/HcalID.h"
+#include "DetDescr/HcalTriggerID.h"
+#include "DetDescr/SimSpecialID.h"
+#include "DetDescr/TrackerID.h"
+#include "DetDescr/TrigScintID.h"
```

Comment:
Is the header necessary? I feel like it would be simpler to just move these lines to the top of the source file (inside the `#if` check).

Reply:
Not really, no.